docs/configuration/monitoring.md

```diff
@@ -107,35 +107,36 @@ helm upgrade --install --atomic --timeout 600s monitoring stable/prometheus-oper
 
 Follow the [Kubernetes documentation](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/access-cluster-services/#accessing-services-running-on-the-cluster) to access the Grafana service that is now running on your cluster, using the method that best suits your use case.
 
-### 5. Enable Service Dashboards
+### 5. Enable service dashboards
 
-Now that you have Prometheus and Grafana installed you can enable `serviceMonitors` and `grafana_dashboards` for Codacy components.
+!!! warning
+    Currently, the application metrics and information reported by the Codacy components to populate service dashboards are very limited since we are still developing these reporting capabilities.
+
+    As such, we recommend that you skip this configuration.
+
+Now that you have Prometheus and Grafana installed you can enable `serviceMonitor` and `grafana_dashboards` for Codacy components.
 
 1.  Create a file named `values-monitoring.yaml` with the following content:
 
     ```yaml
-    codacy-api:
+    website:
       metrics:
         serviceMonitor:
           enabled: true
         grafana_dashboards:
           enabled: true
-    engine:
+    seats:
       metrics:
         serviceMonitor:
           enabled: true
-    worker-manager:
-      grafana:
-        grafana_dashboards:
-          enabled: true
-    listener:
-      grafana:
         grafana_dashboards:
           enabled: true
-    remote-provider-service:
+    rps:
       metrics:
         serviceMonitor:
           enabled: true
+        grafana_dashboards:
+          enabled: true
     ```
```

Comment:
I created this only as a placeholder to provide the correct .yaml to activate service dashboards for the Codacy components that @rtfpessoa reported to be already working: website, seats, and rps.

Who can provide the correct .yaml to perform these configurations?

Reply:
`global:
    metrics:
        grafana:
            enabled: true`

That is the path going forward. :) All components are now tied to it, instead of enabling grafana per component

Reply:
Is this the change that #522 will implement? :thinking:

Reply:
I will add it there. 👍

Reply:
> `global: metrics: grafana: enabled: true`
> 
> That is the path going forward. :) All components are now tied to it, instead of enabling grafana per component

I see, this is great since the solution we provide in the documentation will be forward compatible when we add more supported components. :+1: